### ISSUE TITLE
Styling for mobile?

### ISSUE THREAD

**Issue by kinhbang89** [buildo/react-cookie-banner#13]

**Comment by aquariumtap** [comment]
No, you will have to provide your own. I did this by setting prop **disableStyle** to true, then providing a custom class using prop **className**, which contained my styling and media queries.
